auth-web/src/views/auth/HomeViewDev.vue

```diff
@@ -247,11 +247,16 @@ export default class HomeViewDev extends Vue {
     }
   }
 
-  @media only screen and (max-width: 600px) {
+  @media only screen and (max-width: 640px) {
```

Comment:
Just confirming...
initial value was 600px
new value is 640px
OK?

Reply:
Yessir your bet, should have been 640 initially!

Reply:
Resolved!